shielder/deploy/deploy.sh

```diff
@@ -9,9 +9,9 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
 # bump corresponding tag whenever a new version is released (updates should not be quite via `latest` tag)
 # TODO: Replace with published snarkeling node.
-export NODE_IMAGE=${NODE_IMAGE:-public.ecr.aws/p6e8q1z1/aleph-node-liminal:275670e}
-export CLIAIN_IMAGE=${CLIAIN_IMAGE:-public.ecr.aws/p6e8q1z1/cliain-liminal:8e3643e}
-export CARGO_IMAGE=${CARGO_IMAGE:-public.ecr.aws/p6e8q1z1/ink-dev:1.0.0}
+export NODE_IMAGE="public.ecr.aws/p6e8q1z1/aleph-node-liminal:latest"
```

Comment:
I'm for using fixed tag, especially because we are doing so for `cliain` - this will guarantee no unexpected breakage

Reply:
B/c we don't publish cliain-liminal with `latest` tag - which I think is wrong. We are not releasing liminal-related features any time soon and if we use fixed version we will never detect if we broke it. Who is going to check if this repo is compatible with latest version? 

I can use fixed but I think it's incorrect for this fast-moving context.

Reply:
I'm just concerned with the incoming hackathon

Reply:
Changed in [Use fixed image for aleph-node-liminal.](https://github.com/Cardinal-Cryptography/zk-apps/pull/26/commits/8ba94ddd62cf004e5366b41e035c8f609237eb97)